common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -318,6 +317,8 @@ def load_item(self, location, for_parent=None):  # pylint: disable=method-hidden
                         ).replace(tzinfo=UTC)
                     module._edit_info['published_by'] = raw_metadata.get('published_by')
 
+                module._field_data = OverrideModulestoreFieldData.wrap(module)  # pylint: disable=protected-access
```

Comment:
This takes an `XBlock` and returns a `FieldData`? That's a bit odd. It would be better to keep the input and output consistent (`FieldData` to `FieldData`), or have the function re-inject the `FieldData`. I'd prefer the former.